src/app/datagrid/datagrid-treegrid-dynamicfiltering.demo.html

```diff
@@ -12,11 +12,13 @@ <h2 class="fieldset-title">Soho Data Grid - TreeGrid Dynamic Filter</h2>
   <div class="row top-padding">
     <div class="twelve columns demo" role="main">
       <section class="scrollable contained-scrolling-right-bottom" style="height: 100%">
-        <div soho-busyindicator soho-datagrid
-             selectable="single"
+        <div soho-busyindicator soho-datagrid selectable="single"
+
+             *ngIf="gridOptions"
              [treeGrid]="true"
-             [columns]="columns"
+             [gridOptions]="gridOptions"
```

Comment:
Whats the explanation about this change? I'm wondering why you dont need columns. But have a new gridOptions rather than just using the settings object that matches EP?

Probably a reason im sure but just wondering as it seems inconsistent with the EP API

Reply:
Ok, I see its just part of the sample, i thought it was API. So i think this matters less.